detox/src/devices/drivers/DeviceDriverBase.js

```diff
@@ -108,7 +108,10 @@ class DeviceDriverBase {
   }
 
   createPayloadFile(notification) {
-
+    const fs = require('fs');
```

Comment:
moved from IosDriver

Reply:
Is requiring `fs` so lately caused by unit tests?

Reply:
no - just residue from an intermediate refactor state. thanks for pointing out!